### DISCUSSION TITLE
Submit review pushbutton

### DISCUSSION THREAD

**Discussion by gboydjha** [community/community#63724] [Accessibility] [answered]
### Select Topic Area

Bug

### Body

Submit review pushbutton is not visible when reviewing changes.

I set my screen resolution high because of vision issues.  When reviewing changes, the radio buttons are there for Comment, Approve and Request Changes but the pushbutton for Submit review is not visible.

**Comment by ghostinhershell** [comment]
Hi @gboydjha, thanks so much for reaching out in the Accessibility community 👋🏾 

Where exactly are you running into this issue? It sounds like you're seeing this issue when [reviewing PRs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) but I just want to be sure. Also, if you can send a screenshot of what you're seeing that would be super helpful as well.

Here's instructions on how to take a screenshot depending on whether you're using a Mac or PC: https://reviewed.usatoday.com/laptops/features/how-to-take-a-screenshot-on-mac-and-pc

Cheers!

**Comment by gboydjha** [comment, marked as answer]
Yes, the problem was during pull request approval.  I have since changed my monitor resolution which eliminates the problem.  Here is the screen shot from the original problem.

***@***.***

Gary Boyd | Software Engineer | ARGO Integration | Jack HenryTM
1021 Central Expressway South | Allen, TX 75013 | Cell: 214.458.1389
***@***.***
